modules/imaging_browser/test/imaging_browserTest.php

```diff
@@ -657,43 +664,38 @@ function testImagingBrowserSiteDependingOnPermissions()
      */
     function testImagingBrowserSortableByTableHeader()
     {
-        $this->markTestSkipped(
-            'Skipping tests until Travis and React get along better'
-        );
         $this->setupPermissions(array('imaging_browser_view_allsites'));
         $this->webDriver->navigate()->refresh();
         $this->safeGet(
             $this->url . "/imaging_browser/"
         );
 
-        $PSCIDHeader = $this->safeFindElement(
+        $this->safeClick(
             WebDriverBy::cssSelector(
-                ".info > th:nth-child(3) > a:nth-child(1)"
+                "#dynamictable .info > th:nth-child(3)"
             )
         );
-        $this->clickToLoadNewPage($PSCIDHeader);
 
         $FirstEntry = $this->safeFindElement(
             WebDriverBy::cssSelector(
-                ".table > tbody:nth-child(2) > tr:nth-child(1) > td:nth-child(3)"
+                "#dynamictable tbody > tr:nth-child(1) td:nth-child(3)"
             )
         )->getText();
         $this->assertStringContainsString("AOL0002", $FirstEntry);
 
         // click again and make sure the order is now reversed
-        $PSCIDHeader = $this->safeFindElement(
+        $this->safeClick(
             WebDriverBy::cssSelector(
-                ".info > th:nth-child(3) > a:nth-child(1)"
+                "#dynamictable .info > th:nth-child(3)"
             )
         );
-        $this->clickToLoadNewPage($PSCIDHeader);
 
         $FirstEntry = $this->safeFindElement(
             WebDriverBy::cssSelector(
-                ".table > tbody:nth-child(2) > tr:nth-child(1) > td:nth-child(3)"
+                "#dynamictable tbody > tr:nth-child(1) td:nth-child(3)"
             )
         )->getText();
-        $this->assertStringContainsString("BOL0003", $FirstEntry);
+        $this->assertStringContainsString("ROM300", $FirstEntry);
```

Comment:
It's hard to tell what the correct value is here from the diff or if the change in behaviour was intentional from a PR when the test was skipped or a bug introduced at some point. Can you include a comment in the code explaining the logic of why it's "ROM300"?

Reply:
There are no BOL0003 subjects on RB so not sure why it is in the test here. Unless it was testing for subjects that do not exist?